python/graphscope/client/session.py

```diff
@@ -1299,6 +1306,12 @@ def add_lib(self, resource_name):
         garfile.append("{}".format(resource_name.split("/")[-1]), bytes_)
         self._grpc_client.add_lib(garfile.read_bytes().getvalue())
 
+    def check_version_compatible(self):
```

Comment:
I think it doesn't need to check this. The `k8s_gs_image ` is just an image name, and this change will disable user to pass their custom image.

```python
sess = graphscopes.session("k8s_gs_image="custom_image:tag"")
```